### ISSUE TITLE
Uploading GGUF Models to GCP

### ISSUE THREAD

**Issue by KickItLikeShika** [operatorai/modelstore#285]
are GGUF optimized models supported? 
<img width="1217" alt="Screenshot 2024-09-09 at 2 33 53 PM" src="https://github.com/user-attachments/assets/3e78ca0b-c544-4f59-bb06-9256c00a6fd0">

**Comment by nlathia** [comment]
Is this using huggingface @KickItLikeShika?

It's tricky to keep `modelstore` fully up to date with HF, but one straightforward alternative you can have is to save the file and use modelstore to save the file:
https://modelstore.readthedocs.io/en/latest/concepts/libraries.html#what-to-do-if-a-library-is-not-supported

**Comment by KickItLikeShika** [comment]
Nope, it's using [llama.cpp](https://github.com/ggerganov/llama.cpp), but GGUF formats are supported in HF
thanks for dropping the link, will give it a look

**Comment by nlathia** [comment]
Thanks! Let me know how it works out for you @KickItLikeShika. Perhaps you could consider contributing to the library as well?

https://github.com/operatorai/modelstore/blob/main/modelstore/models/CONTRIBUTING.md
